src/md5/md5.cc

```diff
@@ -219,6 +198,15 @@ void MD5::finalize (){
 
 
 
+MD5::MD5(const unsigned char* input, uint4 input_length) {
+	init();  // must be called be all constructors
```

Comment:
Typo: by

Reply:
I shouldn't have copy-pasted the file-constructor but the stream-constructor, there its correct...